eng/scripts/Copy-Artifacts.ps1

```diff
@@ -0,0 +1,12 @@
+param (
+  [Parameter(mandatory = $true)]
+  $serviceDirectory,
+  [Parameter(mandatory = $true)]
+  $artifactName,
+  [Parameter(mandatory = $true)]
+  $targetPath
+)
+
+New-Item -Type Directory -Name $artifactName -Path $targetPath
```

Comment:
Not that I'm opposed to this but is there much value in having these in their own script as opposed to just inline in the yml file?

Reply:
That's what I was doing initially then I changed to handle all artifacts within this script but array casting from yaml to PS didn't work well. I will make a change to convert this back into inline.